src/device-registry/config/constants.js

```diff
@@ -49,6 +49,8 @@ const stageConfig = {
 };
 
 const defaultConfig = {
+  BIG_QUERY_LOCATION: process.env.BIG_QUERY_LOCATION,
+  TENANTS: process.env.TENANTS.split(","),
   SITES_TOPIC: process.env.SITES_TOPIC,
   DEVICES_TOPIC: process.env.DEVICES_TOPIC,
   LOCATIONS_TOPIC: process.env.LOCATIONS_TOPIC,
```

Comment:
I notice we are not getting the credentials file any where in this config file.
Do we use it as yet?

Reply:
True @123MwanjeMike , great observation!
So this file to configure the local environment of the machine (server) running the application. Just to ensure that the machine can use the BQ client library accordingly/.

So in my `zshr `profile, I have the following line:
`export GOOGLE_APPLICATION_CREDENTIALS="$HOME/Documents/services/airqo-250220-11d2e4a6ead4.json"`